### PR TITLE
RStudio Editor Theme Setup Tutorial upload

### PR DESCRIPTION
**Pull request by Xiaoyang-Song**
PR from Xiaoyang for RStudio Editor Theme Setup Tutorial.

### CODE REVIEW

File: rstudio_editor_theme_setup_tutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+</style>
+
+```{r include=FALSE}
```
Comment (lauradls) [inline rstudio_editor_theme_setup_tutorial.Rmd:22]: Update to {r, include=FALSE}

Comment (Xiaoyang-Song) [reply rstudio_editor_theme_setup_tutorial.Rmd]: Hi, thanks for pointing that out. I just updated it.

**Comment by Xiaoyang-Song** [comment]
Also, I forgot to mention in my previous comment that my tutorial requires the package "rsthemes" from Github source to be installed. But I commented it out in the code chunk and add several comments.

File: rstudio_editor_theme_setup_tutorial.Rmd [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+### Using Themes From "rstheme" Library
+
+As you probably noticed, the available themes are very limited in RStudio so it is natural to use themes from existing R libraries. "rstheme" is a well-known R library consisting of well-designed editor themes developed by [Garrick Aden-Buie](https://github.com/gadenbuie). There is a tutorial for this on Garrick's webpage but I will re-introduce and simplify the steps.
+
+1. Install the package from Github by typing the following command:
+
+```{r}
+# Install the package
+# devtools::install_github("gadenbuie/rsthemes")
+# library(rsthemes)
```
Comment (lauradls) [inline rstudio_editor_theme_setup_tutorial.Rmd:68]: Thanks for pointing it out! Can you also uncomment the library() line?

**Comment by lauradls** [comment]
Thank you for your community contribution! 🎉

**Pull request merged**